core/db/entities.py

```diff
@@ -108,9 +76,34 @@ class Package(db.Model):
     )
     cfo_contact = sqla.orm.relationship("Contact", back_populates="cfo_packages", foreign_keys=[cfo_contact_id])
     m_and_e_contact = sqla.orm.relationship(
-        "Contact", back_populates="m_and_e_packages", foreign_keys=[m_and_e_contact_id]
+        "Contact",
+        back_populates="m_and_e_packages",
+        foreign_keys=[m_and_e_contact_id],
     )
 
+    projects = sqla.orm.relationship("Project", back_populates="package")
+
+
+class Project(db.Model):
+    """Stores Project Entities."""
+
+    __tablename__ = "project_dim"
+
+    id = sqla.Column(GUID(), default=uuid.uuid4, primary_key=True)  # this should be UUIDType once using Postgres
+    project_id = sqla.Column(sqla.String, nullable=False, unique=True)
+    # TODO: should this be unique?
```

Comment:
Might be less messy if we try to put shorter comments on the same line as the variable?

Reply:
Theres a few others like this too

Reply:
I don't mind where they go as the aim is to "do" them ie remove them ASAP. These read better on my screen TBH, so will leave as they are (I'll add a blank line above each to make it clearer). Also, the Git-diff looks cleaner when you remove them if they are not in-line

Reply:
makes sense